src/index.js

```diff
@@ -1,36 +1,67 @@
+
 import 'lodash';
 import './style.css';
+import status from './status.js';
 
 const mainList = document.getElementById('main-list');
 
-const myTasks = [{
-  description: 'Read ES6 Modules',
-  completed: true,
+let myTasks = [{
+  description: 'Understand JavaScript ES6',
+  completed: false,
   index: 0,
 },
 {
-  description: 'Read Webpack',
-  completed: true,
+  description: 'Work on my social media',
+  completed: false,
   index: 1,
 },
 {
-  description: 'List structure project',
+  description: 'Complete To Do List project',
   completed: false,
   index: 2,
```

Comment:
- [ ] You still have some features to be implemented.
![arr4](https://user-images.githubusercontent.com/55635977/131872280-de7f02ce-ab20-4b70-8f8e-5c4bdc9a40bd.png)

Reply:
Hello, I converted the functions to arrow functions but for the last request please check at http://localhost:8081 and check local storage it's working, other than that please explain, thanks.

Reply:
The local Storage works perfectly, however, you need to add checkbox in the UI that make a user to change the status to true/false, and the task object's values for `completed` should be updated upon the user actions

Reply:
Hello, this is To-Do list: interactive list I already did everything as required, I asked my stand-up team and they said I don't need any changes.
Please tell me what do you mean by  "you need to add checkbox in the UI that makes a user to change the status to true/false, and the task's object's values for completed should be updated upon the user actions"
Please explain more I'm stuck here, thanks.